config_parser.cc

```diff
@@ -152,6 +152,8 @@ bool NginxConfigParser::Parse(std::istream* config_file, NginxConfig* config) {
   config_stack.push(config);
   TokenType last_token_type = TOKEN_TYPE_START;
   TokenType token_type;
+  // counter to keep track of blocks
```

Comment:
Comments should be [in sentence case](https://google.github.io/styleguide/cppguide.html#Comment_Style)